src/pages/steps/LocationStep.tsx

```diff
@@ -1188,6 +1183,10 @@ const locationStepConfiguration: StepsConfiguration<'location'> = {
   validation: yup.lazy((value) => {
     const url = window.location.href;
 
+    if (url.includes('organizers') && url.includes('create')) {
+      return yup.object();
```

Comment:
shouldn't the validation be stricter?

Reply:
@simon-debruijn in the the case of "organizer creation" the location is not mandatory that's why we return an object here...